src/platform/windows/display_base.cpp

```diff
@@ -510,6 +413,18 @@ namespace platf::dxgi {
     return false;
   }
 
+  HRESULT
```

Comment:
Should probably copy-paste the explanation from PR explaining why these hooks are needed.

Reply:
Added doxygen and a longer form comment to the body of the function.